string NAME=at

ObjectDrone d
ObjectLocation l

bool truth_value
